src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -2317,6 +2393,11 @@ mgp_error mgp_edge_copy(mgp_edge *e, mgp_memory *memory, mgp_edge **result) {
 
 void mgp_edge_destroy(mgp_edge *e) { DeleteRawMgpObject(e); }
 
+mgp_error mgp_edge_is_deleted(mgp_edge *e, int *result) {
+  *result = IsDeleted(e);
```

Comment:
Same comment about wrap exceptions here as in other parts

Reply:
Added